src/views/Divider.tsx

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
-import { Text, Divider } from 'react-native-elements';
+import React, { useContext } from 'react';
+import { Text, Divider, ThemeContext } from 'react-native-elements';
```

Comment:
`useTheme` looks more nicer

Reply:
Okay, I will see and implement that way. Thanks @arpitBhalla